test/test_helper.rb

```diff
@@ -46,6 +63,28 @@ def metadata_for(topic)
   end
 end
 
+def related_topics_for(topic)
+  metadata = metadata_for(topic)
+  return [] unless metadata
+
+  if metadata["related"]
+    metadata["related"].split(",").map(&:strip)
+  else
+    []
+  end
+end
+
+def aliases_for(topic)
+  metadata = metadata_for(topic)
+  return [] unless metadata
+
+  if metadata["aliases"]
```

Comment:
`return [] unless metadata["aliases"]` maybe? I debated suggesting make `related_topics_for` and `aliases_for` share some code but https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)